kubernetes/delete-weblogic-domain-resources.sh

```diff
@@ -53,6 +56,13 @@ cat << EOF
 EOF
 }
 
+function deleteVoyager {
+  if [ `kubectl get ingresses.voyager.appscode.com -l weblogic.domainName --all-namespaces=true | grep "voyager" | wc -l` -eq 0 ]; then
```

Comment:
need to check whether crd ingresses.voyager.appscode.com exists or not first. otherwise it may throw error here.
use variable VOYAGER_ING_NAME instead of ingresses.voyager.appscode.com.